MPF.Frontend/ViewModels/MainViewModel.cs

```diff
@@ -1213,13 +1213,12 @@ private void CacheCurrentDiscType()
             if (defaultMediaType == MediaType.NONE)
                 defaultMediaType = MediaType.CDROM;
 
-            // If we're skipping detection, set the default value
-            if (Options.SkipMediaTypeDetection)
+            // Skip detection if not using DIC (only DIC needs to know media type)
```

Comment:
I'd rather have the `if` removed entirely. The type is still relevant if the user chooses to change to DIC at some point in the process. We can consider removing the block or gating it behind DIC only in the future.